### DISCUSSION TITLE
Welcome to flutter_dialpad Discussions!

### DISCUSSION THREAD

**Discussion by eopeter** [eopeter/flutter_dialpad#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hardiklakhalani** [comment]
Hi. I'm Hardik from India, using this package to create an app that solves a small privacy issue regarding WhatsApp & other Social Media apps. More coming soon, as it is in development. 🧑‍💻
